### PR TITLE
Backport: Fix DaemonMsgCreateSerializer

### PR DESCRIPTION
**Pull request by johanandren**
Refs #22224

### CODE REVIEW

File: project/MiMa.scala [hunk 1]
```diff
@@ -1169,7 +1169,43 @@ object MiMa extends AutoPlugin {
...
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.WireFormats#PropsData#Builder.addClasses"),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.WireFormats#PropsData#Builder.getClasses"),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.WireFormats#PropsData#Builder.addAllClasses"),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.WireFormats#PropsData#Builder.setClasses"),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.serialization.DaemonMsgCreateSerializer.serialize"),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.serialization.DaemonMsgCreateSerializer.deserialize"),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.serialization.DaemonMsgCreateSerializer.deserialize"),
+        ProblemFilters.exclude[FinalClassProblem]("akka.remote.serialization.DaemonMsgCreateSerializer"),
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.remote.serialization.DaemonMsgCreateSerializer.serialization")
```
Comment (johanandren) [inline project/MiMa.scala]: A bit unsure if this is the way to go, now it's in 2.4-2.5 changes in master, but under 2.4.17 here?

Comment (johanandren) [inline project/MiMa.scala]: MiMa for sure didn't like it.

**Comment by patriknw** [comment]
Don't merge this until we have investigated https://github.com/akka/akka/issues/22366

**Comment by johanandren** [comment]
Ah, forgot that MiMa was already unhappy to start with.

**Comment by johanandren** [comment]
PLS BUILD

**Comment by patriknw** [comment]
What is the status here? I fixed the endian issue and that is included here. So we are good to go?

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by johanandren** [comment]
Yes, should be good to merge.

**Pull request merged**